noah/src/main/resources/population/crawler/population/population/items.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
I have removed `*.pyc`. `__init__.py` is required to make Python treat the directories as containing packages. Find more in this link: https://stackoverflow.com/questions/448271/what-is-init-py-for